gguf-py/gguf/tensor_mapping.py

```diff
@@ -1190,6 +1205,7 @@ class TensorNameMap:
 
         MODEL_TENSOR.V_MM_INP_NORM: (
             "multi_modal_projector.norm",
+            "model.multi_modal_projector.layer_norm", # Intern-S1
```

Comment:
This should be removed as the norm is already mapped to `mlp.0`. This is an artifact from InternVL: https://huggingface.co/OpenGVLab/InternVL3-8B-Instruct/blob/a34d3e4e129a5856abfd6aa6de79776484caa14e/modeling_internvl_chat.py#L79

```suggestion
```